daemon/config/config.go

```diff
@@ -56,6 +56,13 @@ var flatOptions = map[string]bool{
 	"default-ulimits":    true,
 }
 
+// skipValidateOptions contains configuration keys
+// that will be skipped from findConfigurationConflicts
+// for unknown flag validation.
+var skipValidateOptions = map[string]bool{
+	"feature-buildkit": true,
```

Comment:
This should be `buildkit` now?

Reply:
Yes, I thought this was fixed. @AntaresS

Reply:
Or maybe just `features`

Reply:
done